.azure-pipelines/pipelines.yml

```diff
@@ -555,7 +555,7 @@ stages:
   dependsOn: ["precheck"]
   jobs:
   - job: release
-    timeoutInMinutes: 120
+    timeoutInMinutes: 180
```

Comment:
Presumably we'll be able to revert this once caches are warm.

Reply:
I think if this was required once we should leave it, since it could happen at any normal tooling update too then.